provider/cluster/client.go

```diff
@@ -19,7 +22,20 @@ import (
 	mtypes "github.com/ovrclk/akash/x/market/types"
 )
 
-var _ Client = (*nullClient)(nil)
+var (
+	_ Client = (*nullClient)(nil)
+	// Errors types returned by the Exec function on the client interface
+	ErrExec                        = errors.New("remote command execute error")
+	ErrExecNoServiceWithName       = fmt.Errorf("%w: no such service exists with that name", ErrExec)
```

Comment:
Doesn't that do the same thing as `fmt.Errorf` ?

Reply:
~It creates a new `error` that embeds the previous `error` so you can have full access to the wrapped `error` it instead of just figuring it out by parsing strings.~

It looks like [they are the same now](https://golang.org/pkg/fmt/#Errorf)